app/views/availabilities/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="container-fluid">
+  <h1>Schedule Availability</h1>
+  <%= link_to 'Logout', destroy_user_session_path, method: :delete %>
+  <%= form_tag("/availabilities", method: "post") do %>
+    <table class="table table-sm">
+      <tr>
+        <th></th>
+        <% start_time = @start_time %>
+        <% end_time = @end_time %>
+        <% while start_time <= end_time do %>
+          <th colspan="2">
+            <span class="time-label">
+              <%= start_time.strftime("%H:%M") %>
+            </span>
+          </th>
+          <% start_time += 1.hour %>
+        <% end %>
+      </tr>
+      <% @availabilities.each do |availability_day| %>
+        <tr>
+          <td class="dayname"><%= availability_day[0].day %></td>
```

Comment:
Oooh I get what you mean here. In that case, maybe in the controller, instead of having an array of length 7, change it to a dictionary mapping "Sun" "Mon" etc to the array of availabilities.

That way I believe will make a lot of things cleaner.